pkg/apis/pmemcsi/v1alpha1/deployment_types.go

```diff
@@ -108,7 +113,7 @@ type DeploymentStatus struct {
 
 // Deployment is the Schema for the deployments API
 // +kubebuilder:subresource:status
-// +kubebuilder:resource:path=deployments,scope=Namespaced
+// +kubebuilder:resource:path=deployments,scope=Cluster
```

Comment:
This looks like a bug fix. Did the bug have any consequences?

Reply:
This was not an issue as we never used before kubebuilder utils to generate the CRD.